### PR TITLE
Vector Read Backed Phasing

### PR DESCRIPTION
**Pull request by jessicaway**
Adds a cloud version of the read-backed phasing sub-pipeline

### CODE REVIEW

File: tasks/gcp/ReadBackedPhasingTasks.wdl [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      -o ~{output_basename}.phased.vcf \
+      --reference ~{reference.ref_fasta} \
+      ~{subset_vcf} ~{input_bam}
+  }
+
+  runtime {
+    docker: runTimeSettings.whatshap_docker
+    preemptible: runTimeSettings.preemptible_tries
+    cpu: "4"
+    memory: "30 GiB"
```
Comment (alimanfoo) [inline tasks/gcp/ReadBackedPhasingTasks.wdl:68]: This seems high?

**Review by alimanfoo — APPROVED** [review]
LGTM. Just one query regarding memory requirements for whatshap phase task.

File: dockerfiles/WhatsHap/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:bionic-20191029
+
+RUN apt-get update \
+  && apt-get install -y python3-pip \
+  python3-dev \
+  build-essential
+
+RUN pip3 install --upgrade pip \
+    && pip3 install whatshap \
+    && export PATH=$HOME/.local/bin:$PATH
```
Comment (gbggrant) [inline dockerfiles/WhatsHap/Dockerfile:10]: Is there a '.local'?

**Review by gbggrant — APPROVED** [review]

File: tasks/gcp/ReadBackedPhasingTasks.wdl (hunk 1 above)
Comment (gbggrant) [reply tasks/gcp/ReadBackedPhasingTasks.wdl:68]: monitoring script showed it used 11 Gigs of memory for one of the validation samples, running on contig 3L.  So maybe a little high.

Comment (jessicaway) [reply tasks/gcp/ReadBackedPhasingTasks.wdl:68]: We can reduce that, maybe 15?

Comment (gbggrant) [reply tasks/gcp/ReadBackedPhasingTasks.wdl:68]: I just tried cutting it back to 15, but it failed.  Looking closer at the monitoring log, it used as much as (almost) 18 gig - for both the small and the validation sample.  So, I'm leaving it at 30.

**Pull request merged**